tests/pytests/common.py

```diff
@@ -670,31 +670,52 @@ def cmd_assert(env, cmd, res, message=None):
     env.assertEqual(db_res, res, message=message)
 
 # fields should be in capital letters
-def getInvertedIndexInitialSize_MB(fields) -> float:
+def getInvertedIndexInitialSize(fields):
     total_size = 0
     for field in fields:
         if field in ['GEO', 'NUMERIC']:
             block_size = 48
             initial_block_cap = 6
             inverted_index_meta_data = 48
-            total_size += (block_size + initial_block_cap + inverted_index_meta_data) / float(1024 * 1024)
+            total_size += (block_size + initial_block_cap + inverted_index_meta_data)
         elif field not in ['TEXT', 'TAG', 'GEOMETRY', 'VECTOR']:
             raise ValueError(f'Field {field} is not supported')
 
     return total_size
 
+# fields should be in capital letters
+def getInvertedIndexInitialSize_MB(fields) -> float:
+    return getInvertedIndexInitialSize(fields) / float(1024 * 1024)
+
 def check_index_info(env, idx, exp_num_records, exp_inv_idx_size, msg=""):
     d = index_info(env, idx)
     env.assertEqual(float(d['num_records']), exp_num_records, message=msg)
 
     if(exp_inv_idx_size != None):
         env.assertEqual(float(d['inverted_sz_mb']), exp_inv_idx_size, message=msg)
 
+# Iterates items in d1 and compare their keys[value] with d2
+# asserts when a key is missing in d2
+# For simplicity, all values are compared as floats
+def compare_numeric_dicts(env, d1, d2, d1_name="d1", d2_name="d2", msg="", _assert=True):
+    for key, value in d1.items():
+        try:
+            res = float(d2[key]) == float(value)
+            if _assert:
+                env.assertTrue(res, message=msg + " value is different in key: " + key)
+            else:
+                if res == False:
+                    return False
+        except KeyError:
+            if _assert:
+                env.assertTrue(False, message=msg + f" key {key} exists in {d1_name} but doesn't exist in {d2_name}")
```

Comment:
```suggestion
                env.assertContains(key, d2, message=msg + f" key {key} exists in {d1_name} but doesn't exist in {d2_name}")
```

Reply:
assertContains doesn't support message :(

Reply:
We should fix it